### PR TITLE
Proxy support and ability to disable using BITS

### PR DESCRIPTION
**Pull request by daxgames**
Configure proxy support and add optional ability to disable use of bits for downloading files in floppy/_packer_config.cmd

### CODE REVIEW

**Comment by daxgames** [comment]
bump

**Comment by 2xyo** [comment]
@daxgames I have tested your PR with our corpor

File: floppy/01-install-wget.cmd [hunk 1]
```diff
@@ -23,10 +23,17 @@ if exist "%filename%" goto exit0
 echo ==^> Downloading "%WGET_URL%" to "%filename%"
 
 :powershell
+if defined http_proxy (
+    powershell -Command "$wc = (New-Object System.Net.WebClient); $wc.proxy = (new-object System.Net.WebProxy('%http_proxy%')) ; $wc.proxy.BypassList = (('%no_proxy%').split(',')) ; $wc.DownloadFile('%WGET_URL%', '%filename%')" >nul
```
Comment (2xyo) [inline floppy/01-install-wget.cmd]: This configuration works if you download a file over HTTP but not over HTTPS. It fails with the first download of wget with my proxy and it works if I use `%https_proxy%`.

File: floppy/_download.cmd [hunk 2]
```diff
@@ -53,11 +53,18 @@ if not defined PACKER_DEBUG set WGET_OPTS=--no-verbose
 if not errorlevel 1 if exist "%filename%" goto exit0
 
 :powershell
-
-powershell -Command "(New-Object System.Net.WebClient).DownloadFile('%url%', '%filename%')" <NUL
+if defined http_proxy (
+    powershell -Command "$wc = (New-Object System.Net.WebClient); $wc.proxy = (new-object System.Net.WebProxy('%http_proxy%')) ; $wc.proxy.BypassList = (('%no_proxy%').split(',')) ; $wc.DownloadFile('%url%', '%filename%')" >nul
```
Comment (2xyo) [inline floppy/_download.cmd]: Same here with HTTPS

**Comment by fduranti** [comment]
Any news on this?

**Comment by 2xyo** [comment]
Hi @tas50, could you please have a look at this PR ? 

This feature proposed by @daxgames is really interesting as proxies are so common in corporate environment.

**Comment by tas50** [comment]
@2xyo Sorry, but I'm not really involved with this project at this point. We (Chef) moved our windows image development over to our bento project at https://github.com/chef/bento/

**Comment by dragetd** [comment]
Similar this as I mentioned in here: https://github.com/boxcutter/windows/pull/87#issuecomment-467913151

@tas50 I just checked your repository and see that you have very clean and small packer scripts. But they also lack a lot of the different image types, only windows 7, 10, etc. (no 8).

I'd really love to see some a project with solid security principles, clean layout, clean scripts and almost as flexible as boxcutter's version.

Hoping we might be able to squeeze this one into shape. :)

**Comment by daxgames** [comment]
See #179 as a replacement.

**Pull request closed without merging**